aminator/util/linux.py

```diff
@@ -158,6 +158,10 @@ def mount(mountspec):
     if mountspec.fstype:
         if mountspec.fstype == 'bind':
             fstype_flag = '-o'
+            # need to make the bind mounts private so other aminate processes
+            # running in parallel do not automagically effect our mounts
+            # preventing us from umount'ing with "device busy" error
+            options_arg = "--make-private"
```

Comment:
Just be careful here -- if you end up defining a mount option in the config, that will take precedence a few lines below this:
https://github.com/coryb/aminator/blob/7f4409c1c48b33467b5b9ae25745db8673c5cff0/aminator/util/linux.py#L177

(may want to have #L177 append the -o to an existing options_arg if it is not None.